### PR TITLE
Revise README

### PR DESCRIPTION
**Pull request by dmpop**

### CODE REVIEW

File: rmt-helm/README.md [hunk 1]
```diff
@@ -5,44 +5,44 @@ It is tested on K3s but should work on any Kubernetes distribution.
 
 ## Overview
 
-Every component of the stack is deployed in a dedicated container via a
-Helm Chart to ease deployment on top of Kubernetes.
+To deployment on top of Kubernetes easier, each component of the stack is deployed in a dedicated container via a
```
Comment (dirkmueller) [inline rmt-helm/README.md:8]: 'easier' reads weird. This is simply the only possible way to deploy something on kubernetes IMHO.

File: rmt-helm/README.md [hunk 2]
```diff
@@ -5,44 +5,44 @@ It is tested on K3s but should work on any Kubernetes distribution.
...
+configured web server right from the start allows you to target your ingress traffic
+(for RMT) to directly to the server. You don't have to configure ingress for RMT specific
+paths handling, as NGINX is configured to do that.
 
 ## Prerequisites
 
-- a running kubernetes cluster
-- helm (v3) command configured to interact with this cluster
+- a running Kubernetes cluster
+- helm (v3) command configured to interact with the cluster
```
Comment (dirkmueller) [inline rmt-helm/README.md:32]: I think we could omit (v3) here, as helm v2 is really dead meanwhile

File: rmt-helm/README.md [hunk 3]
```diff
@@ -5,44 +5,44 @@ It is tested on K3s but should work on any Kubernetes distribution.
...
 
 ## Custom mandatory values
 
-Some values of this chart do not have any sensible defaults:
-- SCC mirroring credentials, please have a look here for [more information](https://documentation.suse.com/sles/15-SP4/html/SLES-all/cha-rmt-mirroring.html#sec-rmt-mirroring-credentials)
+Certain values of the chart do not have any sensible defaults:
```
Comment (dirkmueller) [inline rmt-helm/README.md:36]: 'any sensible default ' sounds weird. Maybe 'a default value' ?

File: rmt-helm/README.md [hunk 4]
```diff
@@ -5,44 +5,44 @@ It is tested on K3s but should work on any Kubernetes distribution.
...
+- SCC mirroring credentials (refer to [more information](https://documentation.suse.com/sles/15-SP4/html/SLES-all/cha-rmt-mirroring.html#sec-rmt-mirroring-credentials) for more information)
 - list of products to mirror
-- list of products to not mirror
-- DNS name the RMT server should be reachable at
-- Configured [storage](https://kubernetes.io/docs/concepts/storage/)
+- list of products not t mirror
```
Comment (dirkmueller) [inline rmt-helm/README.md:39]: This looks wrong. Typo?

File: rmt-helm/README.md [hunk 5]
```diff
@@ -5,44 +5,44 @@ It is tested on K3s but should work on any Kubernetes distribution.
...
+- DNS name of the RMT server
```
Comment (dirkmueller) [inline rmt-helm/README.md:40]: I think this makes it worse. It's important to mention that this needs to be the externally reachable hostname not the internal name.

File: rmt-helm/README.md (hunk 3 above)
Comment (dmpop) [reply rmt-helm/README.md:36]: If there are no default values, then you're right.

File: rmt-helm/README.md (hunk 4 above)
Comment (dmpop) [reply rmt-helm/README.md:39]: Yes.

File: rmt-helm/README.md (hunk 5 above)
Comment (dmpop) [reply rmt-helm/README.md:40]: Good point. I'll rephrase.

File: rmt-helm/README.md [hunk 6]
```diff
@@ -5,44 +5,44 @@ It is tested on K3s but should work on any Kubernetes distribution.
...
+- configured [storage](https://kubernetes.io/docs/concepts/storage/)
 
-You should fill a custom values file before deploying the chart.
+You must fill a custom values file before deploying the chart.
```
[the remark this answers is not in the document]
Comment (dmpop) [reply rmt-helm/README.md:43]: It's not rigid, it's imperative. You either do it, or you don't.

**Review by dirkmueller — APPROVED** [review]

File: rmt-helm/README.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (dirkmueller) [reply rmt-helm/README.md]: well then its 'Fill a custom values file before deploying the chart' for me. that's imperative. "You must fill" sounds very german to me.

**Pull request merged**